src/Nri/Ui/QuestionBox/V3.elm

```diff
@@ -3,6 +3,7 @@ module Nri.Ui.QuestionBox.V3 exposing
     , id, markdown, actions, character
     , standalone, pointingTo
     , containerCss
+    , setTextToSpeechView
```

Comment:
I wonder if this name should be more generic? It's definitely true that the plan is to use this for TTS, but if it accepts arbitrary HTML, then we're not requiring that "the plan" be followed 😅

Reply:
yeah, I think so!